### PR TITLE
Add `nar` module

### PR DESCRIPTION
**Pull request by h2non**
I've added two packages that could be useful for the community. Cheers

https://github.com/h2non/rocky
https://github.com/h2non/nar

### CODE REVIEW

**Comment by arthurvr** [comment]
Let's see what @sindresorhus thinks but I'd rather see those modules mature a little before adding them to the list.

Please read the contribution guidelines before submitting a PR:

> Make an individual pull request for each suggestion.

> Use the following format: `[package](link) - Description.`

> Don't mention Node.js in the description as it's implied.

> Start the description with a capital and end with a full stop/period.

**Comment by h2non** [comment]
Now I'm following the guidelines strictly. Sorry for that.

Regarding to the package maturity, `nar` has more than 1 year and it could be stable. It has been downloaded more than ~10K times. `rocky` is relatively young, that's true, however I believe that this list should provide potential useful and genuine packages for the community, not just in the sense of consuming packages in a single way, also assisting and inviting the community to improve them if they worth to.

**Comment by sindresorhus** [comment]
`rocky` looks interested and definitely worth including, but I'll echo @arthurvr's sentiment that it needs some time to mature before I want to recommend it. Try submitting it again in a month.

Happy to add `nar` now though :)

**Comment by h2non** [comment]
@sindresorhus Sounds very reasonable.

There's the PR update with `rocky` removed. Thanks

File: readme.md [hunk 1]
```diff
@@ -132,7 +132,6 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [wreck](https://github.com/hapijs/wreck) - HTTP Client Utilities.
 - [http-proxy](https://github.com/nodejitsu/node-http-proxy) - A full-featured HTTP proxy.
 
-
```
Comment (arthurvr) [inline readme.md]: Don't remove this line.

File: readme.md [hunk 2]
```diff
@@ -591,7 +590,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [remote-git-tags](https://github.com/sindresorhus/remote-git-tags) - Get tags from a remote git repo.
 - [dot-prop](https://github.com/sindresorhus/dot-prop) - Get a property from a nested object using a dot path.
 - [onetime](https://github.com/sindresorhus/onetime) - Only run a function once.
-
```
Comment (arthurvr) [inline readme.md]: Don't remove this line :)

File: readme.md [hunk 3]
```diff
@@ -591,7 +590,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
...
+- [nar](https://github.com/h2non/nar) - Creates self-contained executables binaries.
```
Comment (sindresorhus) [inline readme.md]: This sounds better:

> Create self-contained executable binaries.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: don't remove

File: readme.md (hunk 3 above)
Comment (sindresorhus) [inline readme.md]: Don't remove empty lines. There's two empty lines to separate main sections.

**Comment by h2non** [comment]
What about now?

**Comment by arthurvr** [comment]
> What about now?

There's still the removal of one of those lines included.

**Comment by h2non** [comment]
Oh, I see, from the HTTP section :S

**Pull request closed without merging**